### PR TITLE
Display dataframe on landing-page example

### PR DESCRIPTION
**Pull request by GaelVaroquaux**

### CODE REVIEW

**Comment by GaelVaroquaux** [comment]
I'm not sure why the last run did not come with a link to the generated docs, so here it is: https://output.circle-artifacts.com/output/job/a0aeb70b-0cfb-40ed-9b9e-1bdd58f124fc/artifacts/0/doc/index.html

**Comment by GaelVaroquaux** [comment]
Ready for review. Thanks for waiting :)

File: doc/_templates/index.html [hunk 1]
```diff
@@ -111,27 +111,117 @@ <h4 class="card-title">Less data wrangling, more machine learning</h4>
...
-<span class="go">Name: 0, dtype: object</span>
-<span class="go"></span>
+<span class="gp">&gt;&gt;&gt; </span><span class="n">df</span> <span class="o">=</span> <span class="n">dataset</span><span class="o">.</span><span class="n">X</span>
+<span class="gp">&gt;&gt;&gt; </span><span class="n">y</span> <span class="o">=</span> <span class="n">dataset</span><span class="o">.</span><span class="n">y</span>
+<span class="gp">&gt;&gt;&gt; </span><span class="n">df</span></pre>
+        </div>
+      </div>
+    </details>
+<div class="output_subarea output_html rendered_html output_result">
+<div>
```
Comment (jeromedockes) [inline doc/_templates/index.html:130]:
```suggestion
<div style="overflow-x: auto;">
```

(or some other way of making the table scrollable)

**Review by jeromedockes — COMMENTED** [review]
great, thanks! I just have a nitpick

File: doc/_templates/index.html (hunk 1 above)
Comment (jeromedockes) [reply doc/_templates/index.html:130]: also why is the details summary black when we hover over it? 
screenshot_2024-07-02T21:44:57+02:00

**Comment by GaelVaroquaux** [comment]
> +<div style="overflow-x: auto;">

> (or some other way of making the table scrollable)

OK. I'll do it in the CSS though :)

**Comment by GaelVaroquaux** [comment]
> +<div style="overflow-x: auto;"> (or some other way of making the table scrollable)
> OK. I'll do it in the CSS though :)

Done. Let's wait for the render (I did check it on my machine :D )

**Comment by GaelVaroquaux** [comment]
Wait, I just saw a bug in white mode

**Comment by GaelVaroquaux** [comment]
> also why is the details summary black when we hover over it?

That's the bug that I saw :)

**Comment by TheooJ** [comment]
Renders nicely on my phone !

**Comment by TheooJ** [comment]
Just checked the light mode, it becomes black too when I hover the toggle

**Comment by GaelVaroquaux** [comment]
Should be fixed now! (CSS with white and black themes is hard :D )

Good to merge!

**Comment by TheooJ** [comment]
Last thing: do we want « tabular learning » without the « - » ?

**Review by jeromedockes — APPROVED** [review]
LGTM @GaelVaroquaux !

**Comment by GaelVaroquaux** [comment]
> Last thing: do we want « tabular learning » without the « - » ?

I apply the rule that compound adjectives are hyphenated:
https://www.grammarly.com/blog/compound-adjectives/

So, its "I love tabular learning", but "skrub is a tabular-learning package"

**Comment by jeromedockes** [comment]
> "I love tabular learning"

who doesn't

**Pull request merged**